LeadKit/LeadKit/Extensions/Observable/Observable+ToastErrorLogging.swift

```diff
@@ -21,7 +21,7 @@
 //
 
 import RxSwift
-import Toast_Swift
+import ToastSwiftFramework
```

Comment:
please revert this.
`import ToastSwiftFramework` used for Carthage, however it will not work with CocoaPods :(